### PR TITLE
Don't allow users enter decimals for the seed count

### PR DESCRIPTION
**Pull request by constanzauanini**
This require the new web-components version to work

### CODE REVIEW

**Review by charlie-tf — APPROVED** [review]

File: src/components/accession2/edit/CalculatorModal.tsx [hunk 1]
```diff
@@ -150,6 +150,7 @@ export default function CalculatorModal(props: CalculatorModalProps): JSX.Elemen
               onChange={(value) => onChange('subsetCount', value)}
               type='text'
               value={record.subsetCount}
+              disableDecimals={true}
```
Comment (karthikbtf) [inline src/components/accession2/edit/CalculatorModal.tsx:153]: we could use type=number and set the step size to enable/disable decimals?

Comment (constanzauanini) [reply src/components/accession2/edit/CalculatorModal.tsx:153]: I tested stepSize, but allows inputing the ',' and '.' and that is what we don't want. The step just work with the arrows

File: src/components/accession2/edit/CalculatorModal.tsx [hunk 2]
```diff
@@ -150,6 +150,7 @@ export default function CalculatorModal(props: CalculatorModalProps): JSX.Elemen
               onChange={(value) => onChange('subsetCount', value)}
               type='text'
               value={record.subsetCount}
+              disabledCharacters={[',', '.']}
```
Comment (karthikbtf) [inline src/components/accession2/edit/CalculatorModal.tsx:153]: do we need '-' here and the other place?

**Review by karthikbtf — APPROVED** [review]
thanks lgtm

File: src/components/accession2/edit/CalculatorModal.tsx (hunk 2 above)
Comment (constanzauanini) [reply src/components/accession2/edit/CalculatorModal.tsx:153]: Yes, we probably don't want negatives quantities haha

**Review by karthikbtf — COMMENTED** [review]
🎉

**Pull request merged**